kupfer/obj/objects.py

```diff
@@ -440,7 +440,7 @@ def get_description(self):
         return self.object
 
     def get_icon_name(self):
-        return "text-html"
+        return "web-browser"
```

Comment:
UrlLeaf is file (object) not program; so current icon is correct.

Reply:
Ok, I see. But for the bookmarks it is basically just the URL and no local HTML file attached to it. Is there any other way to display that it will be opened by a web browser? I mean you cannot open a URL with a text editor like you could with a .html file. So I think a bookmark URL should not be displayed as html file.